gustaf/io/nutils.py

```diff
@@ -0,0 +1,147 @@
+"""gustaf/gustaf/io/nutils.py
+
+io functions for nutils.
+"""
+import numpy as np
+
+from gustaf.faces import Faces
+from gustaf.volumes import Volumes
+from gustaf.io.ioutils import abs_fname, check_and_makedirs
+from gustaf.utils import log
+from gustaf.io import mixd
+
+
+def load(fname):
+    """nutils load.
+    Loads a nutils (np.savez) file and returns a Gustaf Mesh.
+
+    Parameters
+    -----------
+    fname: str
+      The npz file needs the following keys:
+      nodes, cnodes, coords, tags, btags, ptags.
+    """
+    npzfile = np.load(fname, allow_pickle=True)
+    nodes = npzfile['nodes']
+    _ = npzfile['cnodes']
+    coords = npzfile['coords']
+    _ = npzfile['tags'].item()
+    btags = npzfile['btags'].item()
+    _ = npzfile['ptags'].item()
+
+    vertices = coords
+
+    # connec
+    simplex = True
+    connec = None
+
+    if vertices.shape[1] == 2:
+        volume = False
+    else:
+        volume = True
+
+    try:
+        connec = nodes
+    except BaseException:
+        log.debug("Error")
+
+    # reshape connec
+    if connec is not None:
+        ncol = int(3) if simplex and not volume else int(4)
+        connec = connec.reshape(-1, ncol)
+        mesh = Volumes(vertices, connec) if volume else Faces(vertices, connec)
+
```

Comment:
```suggestion
else:
    raise RuntimeException("Can not generate a mesh from the nutils input. Please check if the nutils mesh description is correct.")

```

Something like this elsewise the return value and the next line `mesh.BC` will definitely fail.